packages/merge-styles/src/Stylesheet.ts

```diff
@@ -142,7 +130,29 @@ try {
 
 let _stylesheet: Stylesheet | undefined;
 
-let constructableStyleSheetCounter = 0;
+const _getGlobal = (win?: Window): typeof _global => {
+  return (win ?? _global) as typeof _global;
+};
+
+const _makeCSSStyleSheet = (win: Window): CSSStyleSheet => {
+  return new (win as Window & typeof globalThis).CSSStyleSheet();
+};
+
+const _cloneCSSStyleSheet = (srcSheet: CSSStyleSheet, targetSheet: CSSStyleSheet): CSSStyleSheet => {
+  for (let i = 0; i < srcSheet.cssRules.length; i++) {
+    targetSheet.insertRule(srcSheet.cssRules[i].cssText);
+  }
+
+  return targetSheet;
+};
+
+const getDocument = () => {
```

Comment:
Not using the `@fluentui/utilities` versions of these functions because that package depends on this one and I don't want to create a circular dependency.